SPECS-SIGNED/mstflint-driver-signed/mstflint-signed.spec

```diff
@@ -0,0 +1,32 @@
+%define mstflint_unsigned_name mstflint-kernel
+%global kver %(/bin/rpm -q --queryformat '%{RPMTAG_VERSION}-%{RPMTAG_RELEASE}' $(/bin/rpm -q --whatprovides kernel-devel))
```

Comment:
This might be problematic, because the specs are parsed twice in our tooling and for this package in both cases the `kernel-devel` package is not going to be present in the build environment, so this will probably fail to do, what we'd like it to do. Let's chat offline, because I think I need to learn more about the expected installation directory for kernel modules.

Reply:
Discussed offline, seems like I was wrong and it's working fine. I'll double-check what the buddy builds return but no need to worry about it for now.